.github/workflows/release.yml

```diff
@@ -47,12 +48,12 @@ jobs:
         roleId: ${{ secrets.VAULT_ROLE_ID }}
         secretId: ${{ secrets.VAULT_SECRET_ID }}
 
-    - name: Publish Docker Image
+    - name: Build/Push Profiler Docker Image
+      continue-on-error: true #continue for now until we see it working in action
       run: |
-        chmod +x .ci/linux/push_docker.sh
-        .ci/linux/push_docker.sh
-      env:
-        TAG_NAME: ${{ github.ref_name }}
+        docker build . -t docker.elastic.co/observability/apm-agent-dotnet:${{ steps.bootstrap.outputs.agent-version }} \
+            --build-arg AGENT_ZIP_FILE=build/output/elastic_apm_profiler_${{ steps.bootstrap.outputs.agent-version }}-linux-x64.zip
+        docker push docker.elastic.co/observability/apm-agent-dotnet:${{ steps.bootstrap.outputs.agent-version }}
```

Comment:
I'd say to use two different steps, one for building and another one for pushing. So it can retry the pushing step if it failed - sometimes docker.elastic.co can be down or failed.

Reply:
done!

Reply:
Although steps can not be retried in isolation from the UI, only jobs.